ha/core/cluster/cluster_manager.py

```diff
@@ -16,16 +16,19 @@
 # cortx-questions@seagate.com.
 
 import time
+import traceback
 
 from cortx.utils.log import Log
+from cortx.utils.conf_store.conf_store import Conf
 from cortx.utils.ha.dm.decision_monitor import DecisionMonitor
 
 from ha.core.error import HAUnimplemented
 from ha.core.node.replacement.refresh_context import PcsRefreshContex
 from ha.execute import SimpleCommand
-from ha.core.support_bundle.ha_bundle import HABundle, CortxHABundle
 from ha import const
 from ha.core.config.config_manager import ConfigManager
+from ha.core.controller.element_controller_factory import ElementControllerFactory
+from ha.core.error import ClusterManagerError
 
 class ClusterManager:
```

Comment:
We need to review names of functions.
Also, the functionality regarding storageset. Probably it's missing in HLD too. Let's discuss this after I am done with other PR too.

Reply:
Added storageset element and its operation

Reply:
Done